From 0482288575960c3c9da99434c7a39a1783b35943 Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Tue, 9 Jun 2020 14:27:00 +0800
Subject: [PATCH] phy: rockchip: inno-video-combo: Initialize in stack
 clk_init_data to 0

This allows adding fields to clk_init_data without having to explicitly
set those fields in all the drivers.

Fixes: 2071154513ec ("ANDROID: GKI: clk: Initialize in stack clk_init_data to 0 in all drivers")
Change-Id: I6d958aa57097209165cf32a671f9612752eec4f1
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 drivers/phy/rockchip/phy-rockchip-inno-video-combo-phy.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/phy/rockchip/phy-rockchip-inno-video-combo-phy.c b/drivers/phy/rockchip/phy-rockchip-inno-video-combo-phy.c
index 378556577823..de5dc3743af6 100644
--- a/drivers/phy/rockchip/phy-rockchip-inno-video-combo-phy.c
+++ b/drivers/phy/rockchip/phy-rockchip-inno-video-combo-phy.c
@@ -723,7 +723,7 @@ static int inno_video_phy_pll_register(struct inno_video_phy *inno)
 	struct device *dev = inno->dev;
 	struct clk *clk;
 	const char *parent_name;
-	struct clk_init_data init;
+	struct clk_init_data init = {};
 	int ret;
 
 	parent_name = __clk_get_name(inno->ref_clk);
-- 
2.35.3

